src/views/Project/index.jsx

```diff
@@ -129,14 +129,14 @@ const tagReposMapping = repositories =>
   seeMoreButton: {
     position: 'absolute',
     left: 3 * theme.spacing.unit,
-    top: 160,
+    top: 184,
   },
   transparentToWhiteBackground: {
-    background: 'linear-gradient(to bottom, transparent 0%, white 50%)',
+    background: 'linear-gradient(to bottom, transparent 0%, white 42%)',
```

Comment:
i saw that the space between the text bottom and button top part is 24 px. To create this spacing, I make sure the fade effect has become white when the vertical distance <= 24px. So the white percentage in the background = (64-24) / 104 = 38% .. Do you think its a valid calculation?
<img width="751" alt="screen shot 2019-02-21 at 7 43 40 pm" src="https://user-images.githubusercontent.com/24544912/53166758-28e5bf00-3611-11e9-8737-143f523b007f.png">